### ISSUE TITLE
[Action] Reach out to FOSDEM for potential collaboration

### ISSUE THREAD

**Issue by leonardpahlke** [cncf/tag-env-sustainability#307]
Action item, part of the Kubecon planning meeting (12th of Dec 2023). We would like to reach out to FOSDEM for potential collaboration. FOSDEM is a conference which is happening in February https://fosdem.org/2024/ (EU, Brussels)

/assign @caradelia

**Comment by guidemetothemoon** [comment]
@caradelia were you able to reach out to FOSDEM or will this be more of a long-term goal? Let us know if there's anything that's needed (help, contact points, etc.) with regards to this issue☺️

**Comment by leonardpahlke** [comment]
@immavalls mentioned that she could reach out next FOSDEM iteration

**Comment by immavalls** [comment]
Talking with @rossf7 this weekend, I thought we should apply for a DevRoom next year. Maybe in collaboration with the Green Software Foundation and other folk. To have a more focused track. The [energy DevRoom](https://archive.fosdem.org/2023/schedule/track/energy/) is mostly about the Energy domain and we could benefit from something more aligned to green software & cloud sustainability. 

I'd love to help coordinate this and apply for a Devroom in 2025. Maybe a half-day room, with fewer talks, to start with. DevRooms can be either half or full days. I'm sure folks will have enough interesting talks to share, with all the ongoing projects, and since it's still 1 year away. The Cloud Sustainability Week was already a success last year, and I'm sure with this we could have an in-person track dedicated to OSS green software with fantastic talks.

**Comment by rossf7** [comment]
@immavalls @guidemetothemoon Yes, I like this idea a lot. The energy dev room did have some green software talks like the one from Boavizta but the focus was more on energy which makes sense. 

We should be able to find enough talks for a half day and probably for a full day and the sustainability talks I attended were all pretty full so I think there would be interest in this.

Adding @mrchrisadams from the Green Web Foundation as we also discussed this. Chris might be able to help with liaising with GSF and ClimateAction.tech which is another org that might be interested in collaborating on this.

**Comment by mrchrisadams** [comment]
Hey folks, I'd totally be up to collaborating on this too. I'll raise it in both venues and see which one responds the most enthusiastically, and with the most support :).

**Comment by immavalls** [comment]
Hi team, can I have this issue assigned to me (and potentially other owners)? I can start work on this in a couple of weeks. There is plenty of time. Applying usually opens in September/October. So we should assemble a team for that in the following months so we are ready when the call for DevRooms opens. I'll put a message on slack asking who wants to join!

**Comment by leonardpahlke** [comment]
Thanks Imma!

**Comment by immavalls** [comment]
Hi everyone, the call for devrooms is open https://fosdem.org/2025/news/2024-09-23-devrooms-cfp/. I am not 100% sure I will be ok to travel to fosdem (still recovering from an injury), so I would propose to apply in 2026. 

If there is enough folks who can make this year though, and manage the room onsite, I can definitely help applying and organizing online. And would join if I am able to travel in February.

Ping me on the cncf slack if it's easier and we can discuss.

**Comment by ChrisChinchilla** [comment]
Hey all! @mrchrisadams and I also spoke about doing this for 2025 and I have run a few devrooms already. So we’re getting this in motion right now.

If any of you are interested in getting involved now (for the CFD) or later on, let me know and let’s collaborate. I will also post this message on the Slack thread that led to this.

**Comment by immavalls** [comment]
Definitely interested if we have a team!

**Comment by locomundo** [comment]
I'd also like to help if we get a devroom :)

**Comment by geurjas** [comment]
Same for me, I'd also like to help if we get a devroom :)

**Comment by ChrisChinchilla** [comment]
Hey all, especially @locomundo and @geurjas as you joined the thread after the rest of us discussed and submitted.

The FOSDEM organisers felt that the pre-existing Energy devroom was close enough already, and they already have more proposals for than space. They suggested we reach out to that room's organisers, which I did.

For 2025, we will instead help them spread the word about their CFP and get more talks we want to see into the dev room, so keep an eye on the FOSDEM announcements. 

And for 2026, we shall see! I think we can probably close this issue now?
